### PR TITLE
feat: add UI improvements

### PR DESCRIPTION
**Pull request by aon**
- Change background to better match zksync page
- Improve loading state for buttons (now with animations!)
- Improve disabled buttons state in proposal page

### CODE REVIEW

File: webapp/app/routes/app/proposals.$id/_route.tsx [hunk 1]
```diff
@@ -163,6 +165,35 @@ export default function Proposals() {
...
+              user.role === "guardian";
+
+            const executeSecurityCouncilApprovalEnabled =
+              securityCouncilSignaturesReached && user.role === "securityCouncil";
+            const executeGuardiansApprovalEnabled =
+              guardiansSignaturesReached && user.role === "guardian";
+            const executeLegalVetoExtensionEnabled =
+              !proposal.extendedLegalVeto &&
+              proposal.status === PROPOSAL_STATES.LegalVetoPeriod &&
+              user.role === "guardian";
```
Comment (hojarasca) [inline webapp/app/routes/app/proposals.$id/_route.tsx:191]: this condition is not needed. Anyone can execute the transaction once the signatures are there.

File: webapp/app/routes/app/proposals.$id/_route.tsx [hunk 2]
```diff
@@ -163,6 +165,35 @@ export default function Proposals() {
...
+            const signLegalVetoEnabled =
+              !userSignedLegalVeto &&
+              !proposal.guardiansExtendedLegalVeto &&
+              proposal.status === PROPOSAL_STATES.LegalVetoPeriod &&
+              user.role === "guardian";
+
+            const executeSecurityCouncilApprovalEnabled =
+              securityCouncilSignaturesReached && user.role === "securityCouncil";
+            const executeGuardiansApprovalEnabled =
+              guardiansSignaturesReached && user.role === "guardian";
```
Comment (hojarasca) [inline webapp/app/routes/app/proposals.$id/_route.tsx:187]: I belive that the veto period have to have ended. The easiest way, I believe, is checking that the state is "Waiting"

File: webapp/app/routes/app/proposals.$id/_route.tsx [hunk 3]
```diff
@@ -163,6 +165,35 @@ export default function Proposals() {
...
+              !userSignedProposal &&
+              proposal.status === PROPOSAL_STATES.Waiting &&
+              (user.role === "guardian" ? !proposal.guardiansApproval : true);
+            const signLegalVetoEnabled =
+              !userSignedLegalVeto &&
+              !proposal.guardiansExtendedLegalVeto &&
+              proposal.status === PROPOSAL_STATES.LegalVetoPeriod &&
+              user.role === "guardian";
+
+            const executeSecurityCouncilApprovalEnabled =
```
Comment (hojarasca) [inline webapp/app/routes/app/proposals.$id/_route.tsx:184]: The upgrade also has to be in "Waiting" state.

**Review by hojarasca — CHANGES_REQUESTED** [review]

File: webapp/app/routes/app/proposals.$id/_route.tsx (hunk 1 above)
Comment (aon) [reply webapp/app/routes/app/proposals.$id/_route.tsx:191]: Execute legal veto can only be called by guardians: https://github.com/matter-labs/zk-governance/blob/654dd0c0b505e3140e7670470356f7edca83f19c/l1-contracts/src/ProtocolUpgradeHandler.sol#L234-L243

File: webapp/app/routes/app/proposals.$id/_route.tsx (hunk 2 above)
Comment (aon) [reply webapp/app/routes/app/proposals.$id/_route.tsx:187]: Fixed!

**Review by hojarasca — APPROVED** [review]

**Pull request merged**